charts/posthog/tests/grafana-dashboards.yaml

```diff
@@ -0,0 +1,61 @@
+suite: ClickHouse instance
+templates:
+  - templates/grafana-dashboards.yaml
+
+tests:
+  - it: should be empty if grafana.enabled is set to false
+    set:
+      grafana.enabled: false
+    asserts:
+      - hasDocuments:
+          count: 0
+
+  - it: the manifest should match the snapshot when using default values
+    set:
+      grafana.enabled: true
+    asserts:
+      - hasDocuments:
```

Comment:
The assert seems to be duplicated across the tests but could just check once? Also, is it redundant the other asserts are successful